crates/vm/levm/src/opcode_handlers/environment.rs

```diff
@@ -341,7 +341,7 @@ impl VM {
         let mut bytecode = self
             .cache
             .get_account(address)
-            .unwrap()
+            .ok_or(VMError::FatalUnwrap)?
```

Comment:
Yes, we were just returning a FatalUnwrap because that in theory cant fail because the code above it caches the account if not cached, even if it has no code. But we can replace it for the `get_account` method in VM and it will work as well

Reply:
https://github.com/lambdaclass/lambda_ethereum_rust/pull/1108/commits/6530ca899e5c2e553d9cb2a9e38d8e2853b7fc5f